client/src/utils/contants.js

```diff
@@ -2,3 +2,4 @@
 
 export const JOB_TYPES = ["Full Time", "Part Time", "Casual", "On-Call"];
 export const STATUS_TYPES = ["Active", "R&I", "Closed"];
```

Comment:
R & I might need to be changed to "R and I" since MySQL doesn't allow ampersands in ENUM values.

Reply:
@1michhu1 this needs to be changed to the following:
```suggestion
export const STATUS_TYPES = ["Active", "Draft", "Inactive"];
```